dlt/dlt.py

```diff
@@ -1072,7 +1080,7 @@ def py_dlt_client_main_loop(client, limit=None, verbose=0, dumpfile=None, callba
         # the status of the callback (in the case of dlt_broker, this is
         # the stop_flag Event), this loop will only proceed after the
         # function has returned or terminate when an exception is raised
-        recv_size = dltlib.dlt_receiver_receive(ctypes.byref(client.receiver), DLT_RECEIVE_SOCKET)
```

Comment:
Maybe we could keep this line

We could add the code block in the beginning of the file

```
from dlt.core import API_VER as API_VER_STR
API_VER = tuple(int(num) for num in API_VER_STR.split('.'))
```

then

```
if API_VER < (2, 18, 6):
    recv_size = dltlib.dlt_receiver_receive(ctypes.byref(client.receiver), DLT_RECEIVE_SOCKET)
else:
    recv_size = dltlib.dlt_receiver_receive(ctypes.byref(client.receiver))
```